.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: os-name
+    attributes:
+      label: OS Name
+      description: Which Operating System are you using?
+      options:
+        - Windows
+        - MacOS
+        - Linux
```

Comment:
While debugging some recent issues with our users, I realized that the majority of the debugging commands we want them to run, as well as some info we want to collect, are different between OSes. We can include the different types of information we want from each OS in this issue template, but I'm afraid that the form will seem too complex/large.

So, my suggestion would be to unroll these OS options into separate issue templates:
* Bug Report (Windows)
* Bug Report (MacOS)
* Bug Report (Linux)

What do you think?

Reply:
That's a great suggestion! I've updated the templates accordingly in 498dcc9152c050e23c23fab562a8db1621e8c90a.